src/Fantomas.Core/CodePrinter.fs

```diff
@@ -3045,30 +3045,26 @@ let genType (t: Type) =
             +> genSingleTextNode node.Closing
 
         let longExpression =
-            let genFields = col sepNln node.Fields genAnonRecordFieldType
-
-            let genMultilineAnonRecordTypeAlignBrackets =
-                let genRecord =
-                    sepOpenAnonRecdFixed
-                    +> indentSepNlnUnindent (atCurrentColumnIndent genFields)
+            let genAnonRecordFields = col sepNln node.Fields genAnonRecordFieldType
+
+            handleBracketStyle (function
```

Comment:
I'm not really a fan of introducing a new helper function with a single usage.
Especially when this usage is essentially doing the same as `ifAlignOrStroustrupBrackets`.

In general, please try to reduce your PR to make the smallest amount of changes.
Before committing, circle back to everything and reflect on whether each change is really necessary to make your point.

Reply:
Yeah sorry, originally I had used this in more than one place but I guess I removed the other usage without realizing it haha